.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,11 +23,11 @@ Fixes #
 I have:
 
 - [ ] Read and followed Crossplane's [contribution process].
-- [ ] Added or updated unit tests.
-- [ ] Added or updated e2e tests, if necessary. (check with reviewers/maintainers if you're unsure whether E2E tests are necessary for the change).
 - [ ] Run `make reviewable` to ensure this PR is ready for review.
-- [ ] Added `backport release-x.y` labels to auto-backport this PR, if necessary.
-- [ ] Opened a PR updating the [docs], if necessary.
+- [ ] Added or updated unit tests.
+- [ ] Added or updated e2e tests.
```

Comment:
with this you have reverted #4805 . The idea behind was to explicitly state that e2e test should added with care, given that they are expensive. I would appreciate if we can get `check with reviewers/maintainers if you're unsure whether E2E tests are necessary for the change` back.

Reply:
This was intentional. I don't really think we need this much context inline of the checklist. Instead in the comment above I've written:

> If you're unsure whether item is needed see the cheat sheet in the contribution docs.

Then, in said cheat sheet I've written:

> If you're still unsure, just leave the checklist box unticked (and un-struck-through). This will cause the `checklist-completed` CI job to fail until you and your reviewer figure out what to do.

Reply:
One minor suggestion is to add a direct link to the cheat sheet that is outside of the markdown comment sections, so folks can find it and view it very easily. Otherwise, I like this streamlining and I like the cheat sheet that gives useful practical guidance and nudges folks to the right direction.